src/command/graph/check.rs

```diff
@@ -29,6 +32,21 @@ pub struct Check {
     #[structopt(long, short = "s", parse(try_from_str = parse_schema_source))]
     #[serde(skip_serializing)]
     schema: SchemaSource,
+
+    /// Minimum number of requests within the time window for a query to be
+    /// considered
+    #[structopt(long, parse(try_from_str = parse_query_count_threshold))]
+    query_count_threshold: Option<i64>,
```

Comment:
this should _maaayyybbbee_ be an `Option<u64>` since I'm assuming you can't (and shouldn't) specify a negative number here. - making it `u64` in that case saves us a bit of space (likely negligible difference in practice, but low-hanging fruit)

Reply:
While that's true, the generated types for the graphql variables for ints are `i64` so I'd still have to cast it back into `i64` at some point, which technically could fail